lib/matplotlib/axes/_axes.py

```diff
@@ -4952,6 +4975,28 @@ def reduce_C_function(C: array) -> float
                 raise ValueError(
                     "y contains non-positive values, so cannot be log-scaled")
             ty = np.log10(ty)
+
+        if xscale == 'symlog' or yscale == 'symlog':
+            symlog_transform = {
+                'x': mscale.SymmetricalLogTransform(
+                    xscale_kwargs.get('base', 10),
+                    xscale_kwargs.get('linthresh', 2),
+                    xscale_kwargs.get('linscale', 1),
+                ),
+                'y': mscale.SymmetricalLogTransform(
+                    yscale_kwargs.get('base', 10),
+                    yscale_kwargs.get('linthresh', 2),
+                    yscale_kwargs.get('linscale', 1),
+                )
+            }
+            inv_symlog_trans = {
+                'x': symlog_transform['x'].inverted(),
+                'y': symlog_transform['y'].inverted()
+            }
+        if xscale == 'symlog':
+            tx = symlog_transform['x'].transform_non_affine(tx)
```

Comment:
```suggestion
            symlog_trafo_x = mscale.SymmetricalLogTransform(
                xscale_kwargs.get('base', 10),
                xscale_kwargs.get('linthresh', 2),
                xscale_kwargs.get('linscale', 1),
            )
            inv_symlog_trafo_x = symlog_trafo_x.inverted()
            tx = symlog_trafo_x.transform_non_affine(tx)
```
And likewise for y below.